2-mandatory/4-stocks.js

```diff
@@ -34,6 +34,19 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
+    let newArr = []
+    let myArr = closingPricesForAllStocks;
+    for (const brand of myArr) {
+        let total = 0;
+        let avg = 0;
+        for (const price of brand) {
+            total += price;
+            avg = total / brand.length
+            avg = Math.round(avg * 100) / 100;
```

Comment:
These 2 lines need to be outside of the loop, as it's value gets overwritten each time the loop runs and only the value after the last run is relevant. 

The code works, but it will be more optimized if you calculate average after the loop.

Reply:
Yes, agreed with @berkeli on this one.

Reply:
@berkeli @moneyinthesky  Oh yea I didn't pay attention to that! Thanks a lot \m/